.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+### What is the feature/fix?
+
+Please summarize the feature or fix.
+
+### What is the Solution?
+
+Summarize what you changed.
+
+### What areas of the application does this impact?
+
+List impacted areas.
+
+# Testing
+
+### Performance Test Results
+
+Please put in details of performance results from testing in WL
```

Comment:
we shouldn't require WL runs to be ran pre-pr. Most tickets won't even need WL runs.

Reply:
That makes sense, I can remove this